server.js

```diff
@@ -1,35 +1,105 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
+import express from "express"
+import cors from "cors"
+import mongoose from "mongoose"
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
 // import avocadoSalesData from "./data/avocado-sales.json";
 // import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
+// import goldenGlobesData from "./data/golden-globes.json"
+import netflixShows from "./data/netflix-titles.json"
+//import { all } from "express/lib/application"
 // import topMusicData from "./data/top-music.json";
 
```

Comment:
You should remove the imported data that are commented out, because you are not using them.